_episodes/21-management.md

```diff
@@ -50,6 +50,23 @@ If you cannot find someone else to bear full responsibility for the Host role, c
 involving helpers in advance of the workshop, or exploring institutional support for event logistics like registration, snack delivery, 
 helper coordination, and emergency contact.
 
+### Co-teaching with other Instructors
```

Comment:
```suggestion
### Instructors: Models for Co-teaching 
```
I agree with your point that the previous title broke the parallelism and I also liked the term `models` in to emphasize that it's multiple examples, not prescriptive.  What about this?